linux-serial-test.c

```diff
@@ -607,16 +644,30 @@ static int diff_ms(const struct timespec *t1, const struct timespec *t2)
 	return (diff.tv_sec * 1000 + diff.tv_nsec/1000000);
 }
 
+static long long int compute_error_count()
```

Comment:
`long long int` seems unnecessary here since  the error count is capped at 125.

Is there a particular reason for this limit?  

```suggestion
static int compute_error_count(void)
```